docs/conf.py

```diff
@@ -213,7 +215,7 @@
     "show_toc_level": 1,
     "use_sidenotes": True,
     "logo": {
-        "text": "Isaac Lab documentation",
+        "text": "Isaac Lab Documentation",
```

Comment:
```suggestion
        "text": f"Isaac Lab v{version}",
```

![image](https://github.com/user-attachments/assets/ab6f3b7f-f097-4bc6-9b69-323447d5e20c)